### PR TITLE
Convert to auto-property

### PR DESCRIPTION
**Pull request by gpetrou**

### CODE REVIEW

File: Src/Microsoft.Dynamic/Debugging/VariableInfo.cs [hunk 1]
```diff
@@ -66,24 +63,18 @@ internal VariableInfo(string name, Type type, bool parameter, bool hidden, bool
         }
 
         /// <summary>
-        /// Type
+        /// Gets the variabl type.
```
Comment (slozier) [inline Src/Microsoft.Dynamic/Debugging/VariableInfo.cs]: type: variable

File: Src/Microsoft.Dynamic/Debugging/VariableInfo.cs [hunk 2]
```diff
@@ -66,24 +63,18 @@ internal VariableInfo(string name, Type type, bool parameter, bool hidden, bool
...
+        /// Gets the name.
         /// </summary>
-        internal string Name {
-            get { return _name; }
-        }
+        internal string Name { get; }
 
         /// <summary>
-        /// Parameter
+        /// Gets or sets a valud indicating whether it is a parameter.
```
Comment (slozier) [inline Src/Microsoft.Dynamic/Debugging/VariableInfo.cs]: typo: value

File: Src/Microsoft.Dynamic/Runtime/ArgumentArray.cs [hunk 3]
```diff
@@ -38,17 +38,14 @@ public sealed class ArgumentArray {
         private readonly int _first;
 
         // the number of items in _arguments that represent the arguments:
```
Comment (slozier) [inline Src/Microsoft.Dynamic/Runtime/ArgumentArray.cs]: Can you move the comment to the property?

File: Src/Microsoft.Dynamic/Ast/SourceFileInformation.cs [hunk 4]
```diff
@@ -20,46 +20,36 @@ namespace Microsoft.Scripting {
     /// source file, in particular the file name and unique language identifier
     /// </summary>
     public sealed class SourceFileInformation {
-        private readonly string _fileName;
-
         // TODO: save storage space if these are not supplied?
```
Comment (slozier) [inline Src/Microsoft.Dynamic/Ast/SourceFileInformation.cs]: Delete the comment.

**Comment by slide** [comment]
This has conflicts now.

File: Src/Microsoft.Dynamic/Actions/Calls/ReturnBuilder.cs [hunk 5]
```diff
@@ -26,16 +26,14 @@
...
-
         /// <summary>
         /// Creates a ReturnBuilder
         /// </summary>
         /// <param name="returnType">the type the ReturnBuilder will leave on the stack</param>
         public ReturnBuilder(Type returnType) {
             Debug.Assert(returnType != null);
 
-            this._returnType = returnType;
+            this.ReturnType = returnType;
```
Comment (slide) [inline Src/Microsoft.Dynamic/Actions/Calls/ReturnBuilder.cs]: nitpick: remove this?

File: Src/Microsoft.Dynamic/Actions/ComboActionRewriter.cs [hunk 6]
```diff
@@ -41,47 +41,29 @@ public class ComboActionRewriter : DynamicExpressionVisitor {
...
-                _type = type;
+                Binders = binders;
+                Inputs = inputs;
+                Type = type;
             }
 
-            public override bool CanReduce {
-                get { return true; }
-            }
+            public override bool CanReduce { get; } = true;
```
Comment (slide) [inline Src/Microsoft.Dynamic/Actions/ComboActionRewriter.cs]: Does this one really need to be changed?

File: Src/Microsoft.Dynamic/Actions/ComboActionRewriter.cs [hunk 7]
```diff
@@ -41,47 +41,29 @@ public class ComboActionRewriter : DynamicExpressionVisitor {
...
 
-            public sealed override Type Type {
-                get { return _type; }
-            }
+            public sealed override Type Type { get; }
 
-            public sealed override ExpressionType NodeType {
-                get { return ExpressionType.Extension; }
-            }
+            public sealed override ExpressionType NodeType { get; } = ExpressionType.Extension;
```
Comment (slide) [inline Src/Microsoft.Dynamic/Actions/ComboActionRewriter.cs]: Does this one really need to be changed?

**Pull request merged**